storage/innobase/dict/dict0stats_bg.cc

```diff
@@ -358,6 +359,7 @@ statistics. */
 void dict_stats_thread() {
   ut_a(!srv_read_only_mode);
   THD *thd = create_internal_thd();
+  thd->wsrep_allow_mdl_conflict = true;
```

Comment:
```create_internal_thd()``` sets ```thd->system_thread = SYSTEM_THREAD_BACKGROUND;```.
Isn't it enough? or do we need to distinguish between background threads?

Reply:
There are many background threads: lots of innodb threads, the wsrep applier, rollbacker, nbo applier threads. I don't think ignoring all of them would be safe, especially that would mean any other new background thread in the future.

So far the conflict only seems to happen because the dict stat thread, nothing else, so it seemed safer to ignore just that one.